src/service/WorkflowService/WorkflowService.Types.ts

```diff
@@ -84,8 +84,8 @@ export namespace WorkflowServiceTypes {
 
     export interface Template {
         templateName: string;
-        title?: string;
-        description?: string;
+        titles: { [key: string]: string };
+        descriptions: { [key: string]: string };
```

Comment:
Please import `Captions` from `@flowfact/types` 

```suggestion
        titles: Captions;
        descriptions: Captions;
```